### PR TITLE
fix: Remove unwanted space in speaker name list

### LINKED ISSUE #2249: If speakers does not add a title it creates unwanted space in speaker description
If a speakers adds both title and company things look fine on the schedule and tracks page. But, if a speaker does not add a job title there is an unwanted space after brackets, e.g. "Daniel Blueman ( FOSSASIA)" instead of  "Daniel Blueman (FOSSASIA)".

Compare below:
Screenshot from 2020-03-18 10-39-57

### PR DESCRIPTION
**Pull request by djcruz93**
Fix: Remove unwanted space after round brackets in speaker name list if speaker position(job title) is not specified

- Add a conditional to check if position is passed

Fixes #2249

### CODE REVIEW

**Comment by djcruz93** [comment]
My second commit is for the next issue #2253

**Comment by djcruz93** [comment]
Do let me know if we need to merge this commit and then raise a separate PR for the next issue.

**Comment by djcruz93** [comment]
@iamareebjamal I have made the changes and raised the PR, whom do I need to add as reviewers?

**Comment by iamareebjamal** [comment]
Before and after screenshots

**Comment by djcruz93** [comment]
**### Fixes #2249 **
**Before the changes**
image

**After the changes**
image

The above changes are fixed in commit #f66f32cade04d4668eae2461a5414cc87220f5d2 (i.e. the first commit)


**### Fixes #2253 ** 
**Before the changes**
image

**After the changes**
image

The above changes are fixed in commit #f747603ff51975d5112e7f127d1be5f029d049cb (i.e. the second commit)

Let me know if I need to create a separate PR for second change. Actually I also did the second change on development branch so it automatically shown in this PR's commits.
@iamareebjamal

File: src/backend/templates/schedule.hbs [hunk 1]
```diff
@@ -221,26 +221,26 @@
...
-                        {{/ifcontains}}
-                        {{#ifcontains audio "aac"}}
-                          <audio controls="" preload="none" id="audio-block">
-                            <source src="{{ audio }}">
-                          </audio>
-                        {{/ifcontains}}
-                        {{#ifvalue description notequals=""}}
-                          <hr class="clear-both">
-                        {{/ifvalue}}
                         <div id="speaker-{{session_id}}" class="session-speakers-list"
```
Comment (mariobehling) [inline src/backend/templates/schedule.hbs:224]: why are those lines moved around? Please only change lines that are related to the issue.

File: src/backend/templates/tracks.hbs [hunk 2]
```diff
@@ -215,23 +215,23 @@
...
-                                  <source src="{{ audio }}">
-                                </audio>
-                              {{/ifcontains}}
-                              {{#ifcontains audio "aac"}}
-                                <audio controls="" preload="none" id="audio-block">
-                                  <source src="{{ audio }}">
-                                </audio>
-                              {{/ifcontains}}
-                              <hr class="clear-both">
                               <div id="speaker-{{session_id}}" class="session-speakers-list" aria-expanded="false"
```
Comment (mariobehling) [inline src/backend/templates/tracks.hbs:218]: why are those lines moved around? Please only change lines that are related to the issue.

**Review by mariobehling — CHANGES_REQUESTED** [review]

File: src/backend/templates/schedule.hbs (hunk 1 above)
Comment (djcruz93) [reply src/backend/templates/schedule.hbs:224]: @mariobehling These lines are moved to resolve issue #2253. I was waiting for this PR(for issue #2366) to get reviewed and in the meantime, I  resolved issue #2253 and made a commit to the development branch, and pushed it to the origin. So the second commit change automatically got added in this PR. Shall I roll back the above change for now and raise another PR after this one gets merged?

File: src/backend/templates/tracks.hbs (hunk 2 above)
Comment (djcruz93) [reply src/backend/templates/tracks.hbs:218]: @mariobehling These lines are moved to resolve issue #2253. I was waiting for this PR(for issue #2366) to get reviewed and in the meantime, I resolved issue #2253 and made a commit to the development branch, and pushed it to the origin. So the second commit change automatically got added in this PR. Shall I roll back the above change for now and raise another PR after this one gets merged?

File: src/backend/templates/schedule.hbs (hunk 1 above)
Comment (mariobehling) [reply src/backend/templates/schedule.hbs:224]: yes, please make the changes in separate PRs. Thanks

File: src/backend/templates/tracks.hbs (hunk 2 above)
Comment (mariobehling) [reply src/backend/templates/tracks.hbs:218]: correct, only make changes in a PR which corresponds to the issue.

**Comment by djcruz93** [comment]
@mariobehling  I have rollbacked the unwanted changes (which were pertaining to a different issue). Can you please check now?

**Review by mariobehling — APPROVED** [review]

**Review by iamareebjamal — APPROVED** [review]
